ob1k-consul/src/main/java/com/outbrain/ob1k/consul/ServiceRegistration.java

```diff
@@ -18,7 +18,11 @@
 
   // I need to make some marshalers happy ;)
   ServiceRegistration() {
-    this(null, 0, null, null, null);
+    name = null;
```

Comment:
The problem was that MsgPack doesn't like the no-empty c'tor even though we don't use MsgPack here at all.
We use JSON here, so this is just a hack basically. The fix is similar to the c'tor, so whatever.